queries/c/highlights.scm

```diff
@@ -103,9 +103,6 @@
 
 (call_expression
   function: (identifier) @function)
-(call_expression
-  function: (field_expression
-    field: (field_identifier) @function))
```

Comment:
@vigoux This can only happen in C++, right? Or should we consider the case of a function pointer that can cause a expression of type `foo.bar()` in C.

Reply:
I better leave that query for that case.

Reply:
`foo.bar()` can happen in C too indeed